### ISSUE TITLE
Release candidate version in readme doesn't work

### ISSUE THREAD

**Issue by lenguyenthanh** [Kotlin/kotlinx-kover#116]
When I follow the setup in readme:
```
plugins {
     id("org.jetbrains.kotlinx.kover") version "0.5.0-RC"
}
```

Gradle tasks always failed. After I downgrade kover to `0.4.0`, everything is working fine.

I think the problem is we need some to add some maven repository in order to use release candidate versions.

**Comment by shanshin** [comment]
Hi,
may you clarify with what error the build crashes?
You may copy the error from Gradle logs

**Comment by lenguyenthanh** [comment]
When I run with  `./gradlew check --debug` it will run forever, here is when it got stucked:

```
2022-01-11T12:40:41.283+0100 [DEBUG] [org.gradle.internal.operations.DefaultBuildOperationRunner] Build operation 'Apply plugin org.gradle.application to project ':app'' completed
2022-01-11T12:40:45.769+0100 [LIFECYCLE] [org.gradle.cache.internal.DefaultFileLockManager] 
2022-01-11T12:40:45.769+0100 [DEBUG] [org.gradle.cache.internal.DefaultFileLockManager] Waiting to acquire shared lock on daemon addresses registry.
2022-01-11T12:40:45.769+0100 [DEBUG] [org.gradle.cache.internal.DefaultFileLockManager] Lock acquired on daemon addresses registry.
2022-01-11T12:40:45.769+0100 [DEBUG] [org.gradle.cache.internal.DefaultFileLockManager] Releasing lock on daemon addresses registry.
2022-01-11T12:40:45.769+0100 [DEBUG] [org.gradle.cache.internal.DefaultFileLockManager] Waiting to acquire shared lock on daemon addresses registry.
2022-01-11T12:40:45.769+0100 [DEBUG] [org.gradle.cache.internal.DefaultFileLockManager] Lock acquired on daemon addresses registry.
2022-01-11T12:40:45.770+0100 [DEBUG] [org.gradle.cache.internal.DefaultFileLockManager] Releasing lock on daemon addresses registry.
2022-01-11T12:40:55.765+0100 [DEBUG] [org.gradle.cache.internal.DefaultFileLockManager] Waiting to acquire shared lock on daemon addresses registry.
2022-01-11T12:40:55.765+0100 [DEBUG] [org.gradle.cache.internal.DefaultFileLockManager] Lock acquired on daemon addresses registry.
2022-01-11T12:40:55.765+0100 [DEBUG] [org.gradle.cache.internal.DefaultFileLockManager] Releasing lock on daemon addresses registry.
2022-01-11T12:40:55.765+0100 [DEBUG] [org.gradle.cache.internal.DefaultFileLockManager] Waiting to acquire shared lock on daemon addresses registry.
2022-01-11T12:40:55.765+0100 [DEBUG] [org.gradle.cache.internal.DefaultFileLockManager] Lock acquired on daemon addresses registry.
2022-01-11T12:40:55.765+0100 [DEBUG] [org.gradle.cache.internal.DefaultFileLockManager] Releasing lock on daemon addresses registry.
2022-01-11T12:41:05.766+0100 [DEBUG] [org.gradle.cache.internal.DefaultFileLockManager] Waiting to acquire shared lock on daemon addresses registry.
2022-01-11T12:41:05.766+0100 [DEBUG] [org.gradle.cache.internal.DefaultFileLockManager] Lock acquired on daemon addresses registry.
2022-01-11T12:41:05.766+0100 [DEBUG] [org.gradle.cache.internal.DefaultFileLockManager] Releasing lock on daemon addresses registry.
2022-01-11T12:41:05.767+0100 [DEBUG] [org.gradle.cache.internal.DefaultFileLockManager] Waiting to acquire shared lock on daemon addresses registry.
2022-01-11T12:41:05.767+0100 [DEBUG] [org.gradle.cache.internal.DefaultFileLockManager] Lock acquired on daemon addresses registry.
2022-01-11T12:41:05.767+0100 [DEBUG] [org.gradle.cache.internal.DefaultFileLockManager] Releasing lock on daemon addresses registry.
```

**Comment by shanshin** [comment]
What logs will there be if you run the build without using previously created daemons?
eg `/gradlew check --debug --no-daemon`

**Comment by lenguyenthanh** [comment]
I have the same result with `./gradlew check --debug --no-daemon`
Maybe I need to clarify my dependencies: I'm using `gradle wrapper 7.3.3` and `Kotlin 1.6.10`.
And my maven repositories:
```
repositories {
  mavenCentral()
  maven { url = uri("https://jitpack.io") }
  maven { url = uri("https://oss.sonatype.org/content/repositories/snapshots")}
}
```

**Comment by shanshin** [comment]
Can you provide a small reproducer project?

**Comment by lenguyenthanh** [comment]
yes, I can. I will create sometime today or tomorrow.

**Comment by lenguyenthanh** [comment]
I don't have time today, so I'll create a sample project over the weekend or next week!

**Comment by lenguyenthanh** [comment]
Hey, I have a sample here: https://github.com/lenguyenthanh/sparrow/pull/4.

**Comment by shanshin** [comment]
Thanks for the reproducer!
Try to apply the plugin in the root project, instead of the `app`, this should solve the problems.

**Comment by lenguyenthanh** [comment]
oh, thanks for the tip. Is this something kover will fixed or is it caused by something else?
If the latter should we add something to Readme? I can create a PR.

**Comment by shanshin** [comment]
We will fix this in the plugin code in the next release.

**Comment by shanshin** [comment]
Fixed in `0.5.0`
